how-to/customize-workspace/client/src/framework/settings.ts

```diff
@@ -52,20 +52,44 @@ export async function getSettings(): Promise<CustomSettings> {
 
 export async function isValid() {
 	logger.info("Validating source of initial settings");
+	const host = window.location.href.toLowerCase();
+	let hostPage = "";
+	const validHost1 = "platform/provider.html";
```

Comment:
Convert to array of possible hosts with a loop for lookup, makes it much more readable, just a `hostPage = possibleHosts.find(h => host.endsWith(h))` could do it

Reply:
done